kubernetes/README.md

```diff
@@ -0,0 +1,23 @@
+Tested on a Minikube cluster (for the moment).
```

Comment:
at the end, it would deserve a short note saying where (IP / port) you can contact the services